patches/chrome-browser-android-intercept_navigation_delegate_impl.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/android/intercept_navigation_delegate_impl.cc b/chrome/browser/android/intercept_navigation_delegate_impl.cc
+index d654b01eb8063d4fc3978d12313b80eb9b8452c8..a199dfc836adfacf65348d6cba0b27046a0d7ce4 100644
+--- a/chrome/browser/android/intercept_navigation_delegate_impl.cc
++++ b/chrome/browser/android/intercept_navigation_delegate_impl.cc
+@@ -24,6 +24,7 @@ class ChromeInterceptNavigationDelegate : public InterceptNavigationDelegate {
+     NavigationParams chrome_navigation_params(navigation_params);
+     chrome_navigation_params.url() =
+         GURL(net::EscapeExternalHandlerValue(navigation_params.url().spec()));
++    IGNORE_IF_PLAY_VIDEO_IN_BROWSER_IS_AVAILABLE(chrome_navigation_params.url())
```

Comment:
@bridiver youtube checking logic is moved to c++.
I think we have two options for adding our logic.
* patching like this.
* replace this file with our owns.

I think patching is better in here because we have to replace whole codes in this file if we override. WDYT?